system/Database/BaseBuilder.php

```diff
@@ -2803,9 +2803,9 @@ protected function _truncate(string $table): string
 	 */
 	public function getCompiledDelete(bool $reset = true): string
 	{
-		$table = $this->QBFrom[0];
-
-		$sql = $this->delete($table, null, $reset);
```

Comment:
Forgive me for being dumb but why is the `$table` removed?

Reply:
The method expects condition on first argument.
https://github.com/codeigniter4/CodeIgniter4/blob/7ca2a7193ae0a109c75fd47233ae954a6db350e0/system/Database/BaseBuilder.php#L2827-L2834

Reply:
Ah yes. See it now. Thanks for that.